src/app/custom-components/header/header.component.ts

```diff
@@ -8,9 +11,20 @@ declare const $: any;
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
+
+  language = EN_KEY.toUpperCase();
+
+  constructor(private translate: TranslateService) {}
+
   ngOnInit(): void {
     $(() => {
       $('.sidenav').sidenav();
     });
   }
+
+  changeLanguage() {
+    const aux = this.language.toLowerCase();
+    this.language = aux === EN_KEY ? ES_KEY.toUpperCase() : EN_KEY.toUpperCase();
```

Comment:
In fact, the aux should go according to the current language. If the site is in English, the link should display: En